scipy/fft/tests/test_basic.py

```diff
@@ -40,17 +40,6 @@ def fft1(x):
     phase = np.arange(L).reshape(-1, 1) * phase
     return np.sum(x*np.exp(phase), axis=1)
 
-
-class TestFFTShift:
-
-    def test_fft_n(self, xp):
-        x = xp.asarray([1, 2, 3], dtype=xp.complex128)
-        if xp.__name__ == 'torch':
-            assert_raises(RuntimeError, fft.fft, x, 0)
-        else:
-            assert_raises(ValueError, fft.fft, x, 0)
-
-
```

Comment:
Pretty sure this class name and placement was a mistake. Moved the test to a more sensible place below, and made it `np_only`. JAX opts to return an empty array when `n=0`, which doesn't seem unreasonable.